src/test/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfigTests.java

```diff
@@ -0,0 +1,58 @@
+package io.jenkins.plugins.artifact_manager_s3;
+
+import java.util.logging.Logger;
+import com.gargoylesoftware.htmlunit.html.HtmlForm;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import com.gargoylesoftware.htmlunit.html.HtmlSelect;
+import com.gargoylesoftware.htmlunit.html.HtmlTextInput;
+import io.jenkins.plugins.artifact_manager_jclouds.JCloudsArtifactManagerFactory;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+import jenkins.model.ArtifactManagerConfiguration;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class S3BlobStoreConfigTests {
+
+    private static final Logger LOGGER = Logger.getLogger(JCloudsVirtualFileTest.class.getName());
+
+    public static final String CONTAINER_NAME = "container-name";
+    public static final String CONTAINER_PREFIX = "container-prefix";
```

Comment:
not actually valid BTW

Reply:
it is, all your artifact files/folder will be container-prefixWHATEVER, it is not a folder but is valid

Reply:
confusing then

Reply:
so, Do we should add always the end "/" or not allow prefix without it? now we warning that the prefix must have a "/", if it is a folder

Reply:
let's leave it for a follow up

Reply:
My preference is to nail down the desired behavior here, which I _think_ is that it must either be empty, or a path ending in `/`.

Reply:
right now, this is the behavior.

Reply:
According to form validation, yes—just pointing out that the presence of a value in the UI test which would trigger a validation warning is confusing to readers.